### PR TITLE
fix(ui): Use consistent Buttons in group OwnedBy

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: static/app/components/group/ownedBy.tsx [hunk 1]
```diff
@@ -68,25 +68,26 @@ function OwnedBy({group, project, organization}: OwnedByProps) {
           />
         </TitleWrapper>
 
-        <ActionsWrapper>
+        <ButtonBar>
```
Comment (scttcper) [inline static/app/components/group/ownedBy.tsx:71]: buttonbar is overkill imo

Comment (evanpurkhiser) [reply static/app/components/group/ownedBy.tsx:71]: isn't this exactly what this is for tho?

Comment (scttcper) [reply static/app/components/group/ownedBy.tsx:71]: button bar is for marking things active? Its got like 100 lines of styles compared to a flexbox with a gap

Comment (evanpurkhiser) [reply static/app/components/group/ownedBy.tsx:71]: What you mean marking as active? ButtonBar is just for a set of buttons that are next to each other no?

Comment (scttcper) [inline static/app/components/group/ownedBy.tsx:71]: needs a gap

**Review by scttcper — APPROVED** [review]

File: static/app/components/group/ownedBy.tsx (hunk 1 above)
Comment (evanpurkhiser) [reply static/app/components/group/ownedBy.tsx:71]: They get too far with gap

**Pull request merged**